index.js

```diff
@@ -231,9 +255,15 @@ function handleEventData(events) {
         }
 
         // add event to DOM
-        let eventContainer = $('.day-first-events');
-        if (startDate.getDate() === 21) {
-            eventContainer = $('.day-second-events');
+        var eventContainer;
```

Comment:
not sure how happy JS will be if we have an event on any other day and it attempts to get nil @colbygallup

Reply:
It'll probably catch fire if that happens. Can you add a filter before `events.forEach` that removes events outside of the date range we care about?

Reply:
Better yet, something like `if (event.getDate() > 21 || event.getDate() < 16) return;` at the start of the `forEach` anonymous function